CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Unreleased
 
-- [...]
+- **[NEW]** Brand new `HighlightSection` component.
```

Comment:
Wondering if it should be BREAKING CHANGE (the HighlightSection component existed but it's all changed)